test/unittests/Microsoft.ServiceFabric.Services.Remoting.Tests/Microsoft.ServiceFabric.Services.Remoting.Tests.csproj

```diff
@@ -6,14 +6,13 @@
     <AssemblyName>Microsoft.ServiceFabric.Services.Remoting.Tests</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
    <TargetFramework>net6.0</TargetFramework>
-   <DocumentationFile>$(OutputPath)\$(AssemblyName).xml</DocumentationFile>
```

Comment:
I'm curious, why is this removed?

Reply:
Good question. I should have mentioned it. Presence of the DocumentationFile property triggers additional compiler warnings about doc comments missing from the public types or methods. I really like these warnings in prod code because it reminds you to document public APIs, but in the test code it's an overkill, and I think it actually encourages bad naming habits by forcing you to supply a comment that duplicates the type or member name. Converting tests from `void` to `async Task` started generating a lot of warnings, which seems completely useless in test code.